engine/api/src/main/scala/pl/touk/nussknacker/engine/api/namespaces/ObjectNaming.scala

```diff
@@ -0,0 +1,10 @@
+package pl.touk.nussknacker.engine.api.namespaces
+
+trait ObjectNaming {
+  def prepareName(originalName: String, namingContext: NamingContext): String
+}
+
+case object DefaultObjectNaming extends ObjectNaming {
```

Comment:
Maybe better will be NoObjectNaming or OriginalObjectNaming?

Reply:
Hm, or maybe `IdentityObjectNaming`?

Reply:
I'm for `IdentityObjectNaming`